scylla-cql/src/errors.rs

```diff
@@ -277,37 +277,6 @@ pub enum WriteType {
     Other(String),
 }
```

Comment:
Typo in commit message: `request: move CqlRequestKind enum to request module `
`belond` -> `belong`